cartridges/custom_cartridge/cartridge/controllers/CheckoutServices.js

```diff
@@ -0,0 +1,231 @@
+'use strict';
+
+/**
+ * @namespace CheckoutServices
+ */
+
+var server = require('server');
+server.extend(module.superModule);
+
+var csrfProtection = require('*/cartridge/scripts/middleware/csrf');
+
+/**
+ * CheckoutServices-PlaceOrder : The CheckoutServices-PlaceOrder endpoint places the order
+ * @name Base/CheckoutServices-PlaceOrder
+ * @function
+ * @memberof CheckoutServices
+ * @param {middleware} - server.middleware.https
+ * @param {category} - sensitive
+ * @param {returns} - json
+ * @param {serverfunction} - post
+ */
+server.replace('PlaceOrder', server.middleware.https, function (req, res, next) {
```

Comment:
We should not need this replace?

Reply:
I replace placeOrder, because i need to extend it with an error message if request to external service is failing